src/app/collective-rewards/rewards/utils/formatMetrics.ts

```diff
@@ -1,16 +1,11 @@
-import { formatCurrency, toFixed } from '@/lib/utils'
-import { formatUnits } from 'viem'
+import { formatCurrency } from '@/app/collective-rewards/utils/formatting/formatter'
 
-export const formatMetrics = (amount: number, price: number, symbol: string, currency: string) => ({
-  amount: `${toFixed(amount)} ${symbol}`,
-  fiatAmount: `= ${currency} ${formatCurrency(amount * price, currency)}`,
+export const formatMetrics = (
+  amount: number | bigint,
+  price: number | bigint,
+  symbol: string,
+  currency: string,
+) => ({
+  amount: `${formatCurrency(amount, symbol) || 0} ${symbol}`,
+  fiatAmount: `= ${currency} ${formatCurrency(Number(amount) * Number(price), currency) || 0}`,
```

Comment:
This could lead to issues, isn't it?

If a pass, a big number and multiply it but another big number, its going to fail if the numbers are big enough, isnt it?

Before it could because we were passing regular numbers, that of course if it was big enough could fail as well, but we were sending the number as decimals.

Reply:
there should be no difference to the current impl, as in runtime they will behave exactly the same